src/components/MovieCard.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+    <router-link to="/movieDetailTest" class="movie-card">
+     <img :src="movie.src" alt="" class="poster"/>
+        <aside class="movie-card-details">
+            <h3>{{ movie.title }}</h3>
+            <div class="button-container">
+              <div class="thumbs">
+                <button name="upVote" v-on:click="emitActivity">👍</button>
+                <button name="downVote" v-on:click="emitActivity">👎</button>
```

Comment:
Emojis are a good placeholder and then I think we should use something like [this](https://fontawesome.com/icons/thumbs-up?style=solid) or [this](https://vuematerial.io/components/icon/) so its a little more professional looking. also we can style after a click with :focus so that the user knows when theyve up or down voted a movie.